apps/fluent-tester/src/FluentTester/theme/ThemePickers.android.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import { View, PickerPropsAndroid } from 'react-native';
+import { View, PickerProps } from 'react-native';
```

Comment:
I think `PickerProps` will also be removed in RN 0.66. Is there a `PickerProps` definition you can use from '@react-native-picker/picker'?